Changed the boolean variable naming convention section with better examples.

Tarek-23 opened the pull request:
Took @mohamedabusrea's suggestion to remove "does" for superfluousness and enriched the section after reading up on the topic.

Tarek-23 committed:
Changed "does" , added "should", and added more examples

mohamedabusrea commented on Readme.md:
````diff
@@ -292,16 +292,20 @@ Directories are camel case (ex. `webapp/controlMethods`)
 ```
 let isValid = true;
+let arePointersNull = false;
+let shouldRenderComponent = true;
````
```suggestion
let shouldRenderComponent = true;
let hasLicense = true;
```

mohamedabusrea commented on Readme.md:
````diff
@@ -292,16 +292,20 @@ Directories are camel case (ex. `webapp/controlMethods`)
+let valid = true;
+let pointersNull = false;
+let renderComponent = true;
 ```
````
You can add this line after the bad example 👇

Note: Avoid boolean variables that represent the negation of a thing. e.g., use isValid instead of isNotValid

Tarek-23 replied:
True! Bear in mind though that we have some places in the code where we do have to have functions named that way such as in the Custom Model Verification module.

mohamedabusrea commented:
Well done @Tarek-23, We should add link for this new section in Variables section (line 34 )

Tarek-23 committed:
Add Mohamed AS

Co-authored-by: Mohamed Abusrea <mohamedabusrea@gmail.com>

Tarek-23 committed:
Add Mohamed AS's example

Co-authored-by: Mohamed Abusrea <mohamedabusrea@gmail.com>

Tarek-23 committed:
More suggestions by MAS

Added link to new section in the index
Added note about naming booleans with negative conditions.

mohamedabusrea commented:
LGTM

mohamedabusrea commented:
Thanks ex-boss for the merge 🤝. Better late than never 😁.
